.github/ISSUE_TEMPLATE/release.md

```diff
@@ -9,23 +9,12 @@ assignees: ''
 
 ## 1. Preparation on Thursday
 
-- [ ] Merge pull requests in the [Ready To Go] column
-- [ ] Include translations
-  <details><summary>Command line instructions:</summary>
-    <pre>
-    <code>
-    git checkout master
-    git pull upstream master
-    tx pull --force
-    git commit -a -m "Update all locales with the latest Transifex translations"
-    git push upstream master
-    </code>
-    </pre>
-  </details>
-- [ ] Create a tag:
-    - `script/tag_release` will auto increment patch version, otherwise
+- [x] Merge pull requests in the [Ready To Go] column
+- [x] Include translations: `script/release/udpate_locales`
+- [x] Create a tag:
+    - `script/release/tag` will auto increment patch version, otherwise
     - `git push upstream HEAD:refs/tags/vX.Y.Z`
-- [ ] [Draft new release]. Look at previous [releases] for inspiration.
+- [x] [Draft new release]. Look at previous [releases] for inspiration.
```

Comment:
I am a bit wary of pre checking these items, technically they aren't done till you run the script. And also I believe the script doesn't draft the release, it just gives you the link to start drafting.

Reply:
Fair point, I was just looking to remove as many clicks as possible from the process.

Although, it's worth noting this template doesn't _create_ the issue, it only opens a draft. So technically it's not marked 'done' until you click 'create' to confirm 😉 .